src/content/docs/guides/manual-installation.mdx

```diff
@@ -7,35 +7,149 @@ import { Code } from '@astrojs/starlight/components';
 import { getLatestVersion } from "@biomejs/version-utils";
 export const version = await getLatestVersion("stable");
 
-Using Biome's standalone CLI binary can be a great choice if you aren't already using Node.js or `npm` (or any other package manager).
-Or in other words, Biome shouldn't be the only reason for you to have a `package.json`.
+You can install Biome to your machine to use like any other command line tool. This can be a great choice
+if you need to run this manually: for instance, if you use a code editor that does not have a Biome plugin,
+or if you intend to use this in shell scripts or another local tool to automate processing of source code
+files, or if you intend to have an AI coding tool execute this tool.
+
+If you are using Biome in a CI/CD pipeline, you should install it directly into the project using a Node.js
+package manager (`npm`, `pnpm`, etc) and abandon this manual option.
```

Comment:
Since this is the page of manual installation, we shouldn't overstep into CI/CD suggestions.

Plus, we already own our own GitHub action and recipes, which doesn't use npm. 

To sum up, this suggestion shouldn't be here.